landavailability/tests/webclient/test_views.py

```diff
@@ -165,7 +165,7 @@ def test_search_view_postcode(self):
             'nearest_broadband_fast': None
         }]
 
-        api_url = '{0}/api/locations/'.format(
+        api_url = '{0}/api/location-search'.format(
             settings.LAND_AVAILABILITY_API_URL)
```

Comment:
I also missed this https://github.com/alphagov/land-availability-api/blob/master/landavailability/api/views.py#L246
during one of the past code reviews, the permissions should be IsAuthenticatedUser for the get() method otherwise we need to give ad admin token to any API user